frontend/components/JoinGraveBtn.tsx

```diff
@@ -648,16 +673,59 @@ export default function JoinGraveBtn({
       );
     } else {
       return (
-        <Button
-          onClick={handleJoin}
-          disabled={loading}
-          mb="10px"
-          fontFamily="Bungee"
-          fontSize="16px"
-          fontWeight="400"
-        >
-          {loading ? 'Processing...' : 'START'}
-        </Button>
+        <>
+          <AlertDialog
+            motionPreset="slideInBottom"
+            leastDestructiveRef={cancelRef}
+            onClose={() => {
+              setLoading(false);
+              onClose();
+            }}
+            isOpen={isOpen}
+            isCentered
+          >
+            <AlertDialogOverlay />
+            <AlertDialogContent>
+              <AlertDialogHeader>Found existing URDs</AlertDialogHeader>
```

Comment:
can we make the title a bit more friendly?
Something like: "Whoops! not so fast!"